index.html

```diff
@@ -162,6 +164,33 @@ <h2>Skills</h2>
           </div>
         </div>
       </section>
+
+      <footer id="footer">
+        <div class="contact-form-container">
+          <h2>Get started</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+          </p>
+          <form action="https://formspree.io/f/xrgjqpek" method="POST" class="contact-form">
+
+            <div class="name-container labeled-element">
+              <label>Your name</label>
+              <input required type="text" maxlength="30" name="name" placeholder="Your name">
+            </div>
+
+            <div class="email-container labeled-element">
+              <label>Your email</label>
+              <input required type="email" name="email" placeholder="Your email">
+            </div>
+
+            <textarea required class="labeled-element" name="message" placeholder="Enter your message here..."></textarea>
```

Comment:
Please ensure that the message input has maxlength="500" attribute according to the requirement.